lib/charms/bind/v0/dns_record.py

```diff
@@ -91,7 +91,7 @@ def _on_config_changed(self, _) -> None:
 
 logger = logging.getLogger(__name__)
 
-DEFAULT_RELATION_NAME = "dns_record"
+DEFAULT_RELATION_NAME = "dns-record"
```

Comment:
Out of curiosity: Is not allowed to use "_" in relation name?

Reply:
Yes, it is, but it's inconsistent with the spec and our other charms, so I've changed it